app/views/local_transaction/results.html.erb

```diff
@@ -55,9 +55,9 @@
   </div>
   <section class="more">
     <div class="more">
-      <%= render "govuk_publishing_components/components/govspeak", {} do %>
-        <%= sanitize(@publication.more_information) %>
-      <% end %>
+      <%= render "govuk_publishing_components/components/govspeak", {
+        content: @publication.more_information.html_safe
+      } %>
     </div>
   </section>
```

Comment:
Looking at the [schema](https://github.com/alphagov/govuk-content-schemas/blob/7d63f4f503c161e7756a867c02b58af4938dde1b/formats/local_transaction.jsonnet#L7-L10) this more_information field does not seem to be a required field. It's probably prudent to put this whole `<section>` element within a conditional to reflect that.

Reply:
Thanks @kevindew - will fix.